pkgs/development/compilers/c3c/default.nix

```diff
@@ -3,6 +3,8 @@
 , fetchFromGitHub
 , cmake
 , python3
+, testers
+, c3c
 }:
 
 llvmPackages.stdenv.mkDerivation rec {
```

Comment:
I believe you can say:

```
llvmPackages.stdenv.mkDerivation (package: rec {
  # snip

  passthru.tests.version = testers.testVersion {
    inherit package;
    command = "${lib.getExe package} --version";
  };
  # snip
})
```

Reply:
I see. I followed [the pattern that I saw the most](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+testVersion+language%3ANix+path%3Apkgs%2F&type=code), but maybe this is better?

Reply:
Re-pinging @numinit for a response

Reply:
package refers to the final attribute set. It's ultimately really a matter of preference and isn't a hard blocker. :-)

Reply:
I see; I'm going to chose to leave it, thank you for the comment :)